content/_help/trouble-signing-in/forgot-your-personal-key._en.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: help
+title: Forgot your personal key
+category: trouble-signing-in
```

Comment:
Is this supposed to be the same content (or very similar) as the set of pages above? Can we provide a link to the other page to avoid duplicating the content?